File: frontend/src/types/outreach.ts
export interface OutreachContact {
  id: string
  job_id: string
  name: string | null
  title: string | null
  email: string | null
  linkedin_url: string | null
  source: string | null
  email_sent: boolean
  email_sent_at: string | null
  email_subject: string | null
  email_body: string | null
  created_at: string
}

File: backend/app/schemas/outreach.py
import uuid
from datetime import datetime

from pydantic import BaseModel


class OutreachContactResponse(BaseModel):
    id: uuid.UUID
    job_id: uuid.UUID
    name: str | None
    title: str | None
    email: str | None
    linkedin_url: str | None
    source: str | None
    email_sent: bool
    email_sent_at: datetime | None
    email_subject: str | None
    email_body: str | None
    created_at: datetime

    model_config = {"from_attributes": True}


class SendEmailRequest(BaseModel):
    subject: str
    body: str
